### ISSUE TITLE
Game doesn't start after patching polish version released by CDProjekt

### ISSUE THREAD

**Issue by TurboSebo** [OldUnreal/UnrealTournamentPatches#1143]
![image](https://user-images.githubusercontent.com/73588307/208701394-1c3dd057-d94d-423f-b12e-abd9185a7f38.png)

There is error which saying, game can't find Function in 'Function Engine.Canvas.CreateFont'

**Comment by SeriousBuggie** [comment]
How named file which you run? It must be UnrealTournament.exe
Try ally patch again.
Also tell us which version of patch you apply - zip, exe?

**Comment by stijn-volckaert** [comment]
This patch probably replaces some of the game's .u files, so it would have to be updated for 469c.

**Comment by NeonKnightOA** [comment]
A polish version creates by CD Projekt? Now THAT got me interested! And I agree with Smirf. There's a clean way of doing localizations, and if files were modified indirectly then it's going to cause mismatching.

**Comment by 0x15d3** [comment]
it happens for me too i used the setup and it doesnt worked tho

**Comment by rmysliwczyk** [comment]
I've found a solution to make this copy/version work! I managed to get the patch working by deleting all of the files that end with .pol and .pol_u from System folder in UT installation. This will remove the Polish localization, but all of the original files are still present.  There is probably some more elegant solution that involves changing some config file to look for the original files instead of those, but it works!

**Comment by TurboSebo** [comment]
> A polish version created by CD Projekt? Now THAT got me interested! And I agree with Smirf. There's a clean way of doing localizations, and if files were modified directly then it's going to cause mismatching.

Yes, CD Projekt (not CDprojekt Red) was polish game publisher. They released many games in Poland with Polish language.

**Comment by TurboSebo** [comment]
> How named file which you run? It must be UnrealTournament.exe Try apply patch again. Also tell us which version of patch you apply - zip, exe?

I tried both

**Comment by TurboSebo** [comment]
> How named file which you run? It must be UnrealTournament.exe Try apply patch again. Also tell us which version of patch you apply - zip, exe?

I tried both. And yes, file names UnrealTournament.exe
![image](https://user-images.githubusercontent.com/73588307/226207947-3afcf9b2-8410-4ebf-bb71-77e140237b24.png)

**Comment by rmysliwczyk** [comment]
> > How named file which you run? It must be UnrealTournament.exe Try apply patch again. Also tell us which version of patch you apply - zip, exe?
> 
> Sorry that I'm replying now, but due to my University I had no time. I tried both. And yes, file names UnrealTournament.exe ![image](https://user-images.githubusercontent.com/73588307/226207947-3afcf9b2-8410-4ebf-bb71-77e140237b24.png)

As per my previous comment. If you still are having issues try the following steps: Edit UnrealTournament.ini in the folder where the executable is located. Change the variable Language from pol to int (it should say Language=int). Install the patch. and run the game. It should start now, albeit in English. If changing the .ini file doesn't work, delete all of the files ending in .pol and .pol_u and then try again.

**Comment by TurboSebo** [comment]
> I've found a solution to make this copy/version work! I managed to get the patch working by deleting all of the files that end with .pol and .pol_u from System folder in UT installation. This will remove the Polish localization, but all of the original files are still present. There is probably some more elegant solution that involves changing some config file to look for the original files instead of those, but it works! Edit: Actually, all you need to do is change Language=pol to Language=int in UnrealTournament.ini in System folder

OK, changing language in .ini file works! Cool, game has started, but unfortunately polish language is no longer available. For me it's not a problem because I understand English.

**Comment by NeonKnightOA** [comment]
> I've found a solution to make this copy/version work! I managed to get the patch working by deleting all of the files that end with .pol and .pol_u from System folder in UT installation. This will remove the Polish localization, but all of the original files are still present. There is probably some more elegant solution that involves changing some config file to look for the original files instead of those, but it works! Edit: Actually, all you need to do is change Language=pol to Language=int in UnrealTournament.ini in System folder

[If only we could have a proper, cleaner Polish localization...](https://github.com/NeonKnightOA/unreal-localization)
